server/routes/updateGoal/updateGoalController.test.ts

```diff
@@ -136,7 +136,7 @@ describe('updateGoalController', () => {
   })
 
   describe('submitUpdateGoalForm', () => {
-    it('should update goal and redirect to overview page given action is submit-form and validation passes', async () => {
+    it('should update goal and redirect to review updated goal page given action is submit-form and validation passes should redirect to the overview page', async () => {
```

Comment:
Minor (probably copy and paste typo 😁 ) - but we don't need the extra `should redirect to the overview page` on the end of the function name)

Reply:
Fixed this duplicated typo